__tests__/Unit/Components/Searchbar/Searchbar.test.tsx

```diff
@@ -0,0 +1,60 @@
+import Searchbar from '@/components/Dashboard/Searchbar';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+let mockFn: jest.Func;
+
+beforeEach(() => {
+    mockFn = jest.fn();
+});
+
+describe('test searchbar component', function () {
+    it('renders the searchbar with appropriate label', function () {
+        render(<Searchbar label="test-label" handleSearch={mockFn} />);
+
+        const label = screen.getByLabelText('test-label');
+        expect(label).toBeInTheDocument();
+    });
+
+    it('checks if we can type into the searchbar', function () {
+        render(<Searchbar label="test-label" handleSearch={mockFn} />);
+
+        const input = screen.getByLabelText('test-label') as HTMLInputElement;
+
+        fireEvent.click(input);
```

Comment:
why do we need to click on input?

Reply:
simulatind user enent to click and type

Reply:
Oops! I missed this one, removed the click event.